server/channels/app/platform/session.go

```diff
@@ -21,7 +21,7 @@ func (ps *PlatformService) ReturnSessionToPool(session *model.Session) {
 		// To avoid a race between clearing the props and accessing it, clear the props maps before returning it to the pool.
 		clear(session.Props)
 		// Also clear the team members slice to avoid a similar race condition.
-		clear(session.TeamMembers)
+		session.TeamMembers = session.TeamMembers[:0]
```

Comment:
@hanzei - I thought we decided on removing the TeamMembers clearing entirely?

Reply:
Applied the change